hubspot/management/commands/configure_hubspot_bridge.py

```diff
@@ -218,8 +218,8 @@
             },
             {
                 "propertyName": "highest_education",
-                "targetHubspotProperty": "highest_education",
-                "dataType": "NUMBER",  # (or "STRING")
+                "targetHubspotP`roperty": "highest_education",
```

Comment:
typo.  After fixing and testing this locally, the hubspot configuration command worked.